apps/network-alerts/js/attention/attention.js

```diff
@@ -31,26 +30,24 @@ function sendNotification() {
   }
 
   return new Promise(function(resolve, reject) {
```

Comment:
Actually, it would be good to use the opportunity with a small patch like this to clean up the code as much as we can.